.github/actions/e2e-stencil/action.yml

```diff
@@ -12,7 +12,7 @@ runs:
       name: Run Cypress
       with:
         working-directory: ./packages/samples/stencil
-        start: npm start
+        start: npm run dev
```

Comment:
Run dev is typically at the top to run only atomic along w/ headless, perhaps you're not using it, but we should keep it consistent with before?

Reply:
I removed `dev` at the root `package.json`. Trying to run `npm run dev` should give an error, which should prompt people to look at the `package.json` and find `dev:atomic`. I think it's fine.

Reply:
Alright! I do like the coherence